specs/mrtcore/PrimaryLanguageOverride/PrimaryLanguageOverride.md

```diff
@@ -0,0 +1,190 @@
+property.
+
+# Remarks
+
+When language resource packages are used, the packages that are installed and registered for a given
+user are determined by the languages in the user's language profile. The set of languages returned
+by the ManifestLanguages property is limited to the languages currently available on the system for
+the user. Languages included in the main app package manifest will always be returned; languages
+from resource packages will be returned only if the language is applicable for the user (that is, is
+in the user's preferences) and the resource package has been installed and registered for the user
+at the time the property is accessed.
+
+## ApplicationLanguages.PrimaryLanguageOverride property
+
+Gets or sets an override for the app's preferred language, expressed as a
+[BCP-47 language tag](https://tools.ietf.org/html/bcp47). For packaged apps, this property is a
+wrapper for
+[Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride)
+property and behaves the same. This setting is global for the running process. Unlike for packaged
+apps, for un-packaged apps, this setting is not persisted between app sessions.
+
+```c#
+public static string PrimaryLanguageOverride { get; set; }
+```
+
+# Property Value
+
+[String](https://learn.microsoft.com/dotnet/api/system.string)
+
+A [BCP-47 language tag](https://tools.ietf.org/html/bcp47). The app can set it to override the
+language. It must be a single language tag; a delimited list of language tags will result in default
+language being used.
+
+The PrimaryLanguageOverride property should only be set to languages available for the app.
+
+When your app gets the value, PrimaryLanguageOverride returns either a single language tag (if your
+app has previously set the property) or an empty string.
+
+# Remarks
+
+Apps normally run with language settings determined by the system by comparing the languages
+supported by the app with the language preferences of the user. The PrimaryLanguageOverride property
+is used to override that behavior by setting a single specific language.
+
+For packaged apps, `PrimaryLanguageOverride` setting is a wrapper for
+[Windows.Globalization.ApplicationLanguages.PrimaryLanguageOverride](https://learn.microsoft.com/uwp/api/windows.globalization.applicationlanguages.primarylanguageoverride)
+property and behaves the same. It is persisted between app sessions.
+
+For un-packaged apps, `PrimaryLanguageOverride` setting is not persisted between sessions. It should
+be set each time the app is loaded. You should set PrimaryLanguageOverride setting in early stage of
+app loading, before any resource is loaded.
+
+The property should only be set based on user input presented in settings UI. It can be read at any
```

Comment:
'should'. And what if it isn't?

Reply:
This line should be moved to "packaged apps" part, after "It is persisted between app sessions" part. It means that it should not be set on every app launch, but only on user input from app settings.

Reply:
Move to "packaged apps" section.